frontend/src/posapp/components/navbar/AboutDialog.vue

```
@@ -91,70 +91,74 @@
-			appInfoError: false,
-			appInfo: [],
-			buildVersion: BUILD_VERSION,
-		};
-	},
-	computed: {
-		formattedBuildVersion() {
-			return this.buildVersion ? formatBuildVersion(this.buildVersion) : null;
-		},
+});
+
+const BUILD_VERSION = typeof __BUILD_VERSION__ !== "undefined" ? __BUILD_VERSION__ : null;
```

ESLint

[error] '__BUILD_VERSION__' is not defined.

(no-undef)